spec/features/draws/draw_start_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Draw start' do
+  let(:draw) { FactoryGirl.create(:draw, status: 'draft') }
+
+  context 'as admin' do
+    before { log_in(FactoryGirl.create(:admin)) }
+
+    it 'can be started' do
+      visit draw_path(draw)
+      click_link('Begin draw process')
+
+      expect(page).to have_css('.flash-success', text: 'Draw invitations sent')
+    end
+  end
+
+  context 'as student' do
+    before { log_in(FactoryGirl.create(:student)) }
+
+    it 'cannot see the start button' do
+      visit draw_path(draw)
+
+      expect(page).not_to have_link('Begin draw process')
+    end
+  end
```

Comment:
I'm hesitant to leave these permission-based feature specs long-term. I think they're good when behavior differs greatly between roles, but might be overkill for things like seeing a button. 

Perhaps we want to have some view specs instead?